rancher2/structure_project.go

```diff
@@ -186,6 +190,10 @@ func expandProjectResourceQuotaLimit(p []interface{}) *managementClient.Resource
 		obj.ConfigMaps = v
 	}
 
+	if v, ok := in["extended"].(map[string]string); ok && len(v) > 0 {
```

Comment:
This won't work... `TypeMap` comes out as a `map[string]interface{}` https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema#Schema

In this case, the `extended` element field will never be populated.

Reply:
Believed fixed with https://github.com/rancher/rancher/pull/52773